binstar_client/utils/projects/__init__.py

```diff
@@ -0,0 +1,102 @@
+import inspect
+import logging
+import os
+from os.path import basename, isdir, join, relpath
+from binstar_client.errors import BinstarError
+from .filters import filters
```

Comment:
why? what if we extract projects into a new library? why don't you like relative imports?